ext/tags.go

```diff
@@ -83,22 +81,34 @@ var (
 	HTTPStatusCode = uint16TagName("http.status_code")
 
 	//////////////////////////////////////////////////////////////////////
-	// Error Tag
+	// Error Tag name
 	//////////////////////////////////////////////////////////////////////
 
 	// Error indicates that operation represented by the span resulted in an error.
 	Error = boolTagName("error")
 )
 
+var (
+	//////////////////////////////////////////////////////////////////////
+	// Conventional SpanKind Tags
+	//////////////////////////////////////////////////////////////////////
+
+	// TagSpanKindRPCClient is a tag indicating the span is a RPC client.
+	TagSpanKindRPCClient = opentracing.Tag{Key: string(SpanKind), Value: SpanKindRPCClientTagValue}
+
+	// TagSpanKindRPCServer is a tag indicating the span is a RPC client.
+	TagSpanKindRPCServer = opentracing.Tag{Key: string(SpanKind), Value: SpanKindRPCServerTagValue}
```

Comment:
what do we lose if we just call it SpanKindRPCServer?  It matches with the other tags in this file, e.g.
```go
ext.SamplingPriority.Set(span, 1)
ext.SpanKindRPCClient.Set(span) // value implied
```